etc/scripts/docenizer.py

```diff
@@ -31,18 +34,31 @@
     'MOVQ',  # defined in MOVD:MOVQ
     'MOVSD'  # defined in MOVS:MOVSB:MOVSW:MOVSD:MOVSQ
 ]
+# Where to extract the asmdoc archive.
+ASMDOC_DIR = "asm-docs"
 
 
 class Instruction(object):
     def __init__(self, name, names, tooltip, body):
         self.name = name
+        self.url = "http://www.felixcloutier.com/x86/{}.html".format(urllib.quote(name))
```

Comment:
Isn't this storing a bunch of unnecessary duplicates of mostly the same? Wouldn't it be better to keep the url only in one place (L#204)?

Reply:
I now see that this is probably for possible future tests, but I still this should not be here. What about  extracting it to a function and using that function in L#204 *and* in the tests?

Reply:
Indeed, my bad. That's a relic of a refactoring that should have been removed. Good catch! :eye:

Reply:
How would you use the function in the tests? Would you issue a request for every file? As of now, I check all files locally.

Reply:
Yes. That's the only thing we can do right now. If @zneak had a sitemap, we could prob use it to only have to visit one page, but thats just an idea

Reply:
I did not realize this! This is great. @ethiraric do you think you can use those links in the index.html to actually check the urls? Maybe we dont need to create the url ourselves, but use the mapping in index.html

Reply:
That should be doable. I'll work on it in the next few days.